src/helpers/deep_value.js

```diff
@@ -18,7 +18,7 @@ const deepValue = (obj, path, list) => {
     
     if (value !== null && value !== undefined) {
       if (!remaining && (typeof value === 'string' || typeof value === 'number')) {
-        list.push(value)
+        list.push(value.toString())
```

Comment:
this is a fairly breaking change since the results don't come back as numbers any longer